app/helpers/algolia_query_helper.rb

```diff
@@ -0,0 +1,11 @@
+module AlgoliaQueryHelper
+  include Rails.application.routes.url_helpers
+
+  def link_filter_providers(slug)
```

Comment:
Vu que ça produit un path, finir par `_path` me semble plus accurate.

```suggestion
  def endpoints_with_provider_query_filter_path(slug)
```

Reply:
D'ailleurs je passerai le endpoint plutôt que le slug

Reply:
:+1:  

je me permet de merger du coup après correction